src/Models/DomainVerification.php

```diff
@@ -3,8 +3,9 @@
 namespace SunAsterisk\DomainVerifier\Models;
 
 use Illuminate\Database\Eloquent\Model;
+use SunAsterisk\DomainVerifier\Contracts\Models\DomainVerifiableInterface;
 
-class DomainVerification extends Model
+class DomainVerification extends Model implements DomainVerifiableInterface
```

Comment:
`DomainVerification` is not a `DomainVerifiable`.
- `DomainVerifiable` means things which need to verify domain ownership.
- `DomainVerification` is an ownership verification for the `DomainVerifiable`